src/App.jsx

```diff
@@ -0,0 +1,18 @@
+import Home from "./pages/home/Home";
+
+import Footer from "./components/footer/Footer";
```

Comment:
folder for component should have name from upper case. it related to all components